Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7758.xaml.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+    [Preserve(AllMembers = true)]
+    [Issue(IssueTracker.Github, 7758, "[iOS] EmptyView is not rendered in screen center", PlatformAffected.iOS)]
+    public partial class Issue7758 : TestContentPage
+    {
+        public Issue7758()
+        {
+#if APP
+			Device.SetFlags(new List<string> { CollectionView.CollectionViewExperimental });
+
+			InitializeComponent();
+#endif
+		}
+
+		protected override void Init()
```

Comment:
Can you fix the code formatting please?

Reply:
Should be fixed. I think the formatting errors were introduced with copy paste from the other test.